base/process.jl

```diff
@@ -31,42 +31,37 @@ struct Cmd <: AbstractCmd
-  or escaping of arguments, even arguments containing spaces.  (On Windows,
-  arguments are sent to a program as a single "command-line" string, and
-  programs are responsible for parsing it into arguments.  By default,
-  empty arguments and arguments with spaces or tabs are quoted with double
-  quotes `"` in the command line, and `\\` or `"` are preceded by backslashes.
-  `windows_verbatim=true` is useful for launching programs that parse their
-  command line in nonstandard ways.)  Has no effect on non-Windows systems.
-* `windows_hide::Bool`: If `true` (defaults to `false`), then on Windows no
-  new console window is displayed when the `Cmd` is executed.  This has
-  no effect if a console is already open or on non-Windows systems.
-* `env`: Set environment variables to use when running the `Cmd`.  `env`
-  is either a dictionary mapping strings to strings, an array
-  of strings of the form `"var=val"`, an array or tuple of `"var"=>val`
-  pairs, or `nothing`.  In order to modify (rather than replace)
-  the existing environment, create `env` by `copy(ENV)` and then
-  set `env["var"]=val` as desired.
+    Cmd(cmd::Cmd; ignorestatus, detach, windows_verbatim, windows_hide, env, dir)
+
+Construct a new `Cmd` object, representing an external program and arguments, from `cmd`,
+while changing the settings of the optional keyword arguments:
+
+* `ignorestatus::Bool`: If `true` (defaults to `false`), then the `Cmd` will not throw an
+  error if the return code is nonzero.
+* `detach::Bool`: If `true` (defaults to `false`), then the `Cmd` will be run in a new
+  process group, allowing it to outlive the `julia` process and not have Ctrl-C passed to
+  it.
+* `windows_verbatim::Bool`: If `true` (defaults to `false`), then on Windows the `Cmd` will
+  send a command-line string to the process with no quoting or escaping of arguments, even
+  arguments containing spaces. (On Windows, arguments are sent to a program as a single
+  "command-line" string, and programs are responsible for parsing it into arguments. By
+  default, empty arguments and arguments with spaces or tabs are quoted with double quotes
+  `"` in the command line, and `\\` or `"` are preceded by backslashes.
+  `windows_verbatim=true` is useful for launching programs that parse their command line in
+  nonstandard ways.) Has no effect on non-Windows systems.
+* `windows_hide::Bool`: If `true` (defaults to `false`), then on Windows no new console
+  window is displayed when the `Cmd` is executed. This has no effect if a console is
+  already open or on non-Windows systems.
+* `env`: Set environment variables to use when running the `Cmd`. `env` is either a
+  dictionary mapping strings to strings, an array of strings of the form `"var=val"`, an
+  array or tuple of `"var"=>val` pairs, or `nothing`. In order to modify (rather than
+  replace) the existing environment, create `env` by `copy(ENV)` and then set
+  `env["var"]=val` as desired.
```

Comment:
In addition to modifying the wrapping I removed several double spaces after a period.